elements/pfe-textinput/README.md

```diff
@@ -0,0 +1,60 @@
+# PatternFly Element | Textinput element
+A web component text input with support for the following types
+- text
+- password
+- email
+- search
+- tel
+- url
+
+## Usage
+```html
+<pfe-textinput>
+  <input type="text" name="first-name">
+</pfe-textinput>
+```
+
+You can add any of the [standard input attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#Attributes) on the `input` tag inside `pfe-textinput`.
+
+```html
+<pfe-textinput>
+  <input type="text" name="first-name" placeholder="First Name" required>
+</pfe-textinput>
+```
+
+## Slots
+- `default`: It's required that you put an `input` element in the default slot for `pfe-textinput`. 
+
+## Attributes
+- `pfe-c-novalidate`: The presence of this attribute removes any of the internal event listeners (focus, blur, invalid, change, paste, keyup) and, as a result, none of the validity checks will be peformed on this element.
+
+You can add any of the [standard input attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#Attributes) on the `input` tag inside `pfe-textinput`.
+
+## Events
+Listen for any of the standard input events that are available to `input` elements.
+
+## Dev
+
+    `npm start`
```

Comment:
This will render unnecessary backticks.  Try using either backticks or indentation, never both.  This same issue is present [elsewhere](https://github.com/patternfly/patternfly-elements/blob/master/elements/pfe-button/README.md) in the repo as well, and should be cleaned up.

Here's some markdown code block examples on the left, and how they're rendered on the right:

![Screenshot from 2020-09-01 11-19-17](https://user-images.githubusercontent.com/364615/91870617-04769400-ec45-11ea-947a-704da7faf593.png)